01-Introduction.Rmd

```diff
@@ -1,8 +1,83 @@
 # Introduction
 
-This guide is heavily influenced by the [Python Developer Guide](https://devguide.python.org/), and is a comprehensive resource for contributing to R Core – for both new and experienced contributors. It is maintained by the [R Contribution Working Group](/working-group). We welcome your contributions to R Core!
+This guide is heavily influenced by the [Python Developer Guide](https://devguide.python.org/), and is a comprehensive resource for contributing to base R – for both new and experienced contributors. It is maintained by the [R Contribution Working Group](/working-group). We welcome your contributions to base R!
 
-## How to contribute to this guide? {#how-to-contribute-to-this-guide -}
+## Introduction to Bugzilla
```

Comment:
I don't think the introduction to Bugzilla fits in this chapter as it is only relevant to people working on bugs - it may not even be necessary for people that only want to report a bug as they could do that on the R-devel mailing list.

I suggest moving this section to the reviewing bugs chapter, as the bug tracking chapter is already quite long and has direct links to Bugzilla where necessary (e.g. in item 4 in this section: https://contributor.r-project.org/rdevguide/bug-tracking.html#RCorePkgBug). I think we could add a Quick Overview of Bugzilla as a subsection before or after this one: https://contributor.r-project.org/rdevguide/reviewing-bugs.html#preparing-to-review-bug-reports, since getting familiar with the different search options is probably most helpful to people that want to contribute to bug review. We could add a link to this Quick Overview of Bugzilla in the Bug Tracking chapter as well (ch 3) to help people navigate Bugzilla without having to come back to the direct links. (Navigating Bugzilla could be an alternative title for the subsection).